src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -249,6 +267,7 @@ public Result<Void> openLoanHasUserId() {
 
   public Result<Void> closedLoanHasCheckInServicePointId() {
     if (isClosed() && getCheckInServicePointId() == null) {
+      log.warn("closedLoanHasCheckInServicePointId:: a closed loan must have a checkin service point");
```

Comment:
Can we extract message into a variable and reuse it?

Reply:
done